scripts/dev.cjs

```
@@ -0,0 +1,90 @@
+ * piped and prefixed.
+ */
+const { spawn } = require("node:child_process")
+const path = require("node:path")
+const readline = require("node:readline")
+const { clearTimeout, setTimeout } = require("node:timers")
+
+const CONVEX_PREFIX = "\u001b[36m[convex]\u001b[0m "
+const FORCED_KILL_DELAY_MS = 4000
+const localBin = (name) => path.join(process.cwd(), "node_modules", ".bin", name)
+
+// Detached so convex leads its own process group, which shutdown can signal as a
```

ESLint

[error] Prefer self-explanatory code over this comment. Rewrite the code so its intent is expressed through the API, variables, function names, and parameters. Simplify or extract the logic first; keep a comment only when the code cannot express a necessary constraint or rationale.

(self-explanatory-code/prefer-self-explanatory-code)